### DISCUSSION TITLE
Can this application be installed in Linux machine?

### DISCUSSION THREAD

**Discussion by thomasjsln** [Open-Systems-Pharmacology/Forum#797] [Questions & Problems] [answered]
Hi Team,

I saw that tar.gz setup file was available for this application.

Please confirm whether the application can be installed in Linux machine, If so please send me the steps to install.

As far as i see the instructions are provided only for Window machine.  Please confirm!

Best Regards,
Thomas

**Comment by msevestre** [comment, marked as answer]
Hi @thomasjsln 

The OSPSuiste is a windows application only. However, you can export your model to pkml and perform calculations etc.. using our R toolbox which is compatible with Linux.

See [Installation for Linux](https://github.com/Open-Systems-Pharmacology/OSPSuite-R#under-linux)

See [Working with R](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction)

Hope that helps,

**Comment by thomasjsln** [comment]
Hi Michael,

Thank you for the information.

This really helps!

Regards,
Thomas
